banking_system/banking_system/settings.py

```diff
@@ -26,7 +26,7 @@
 DEBUG = True
```

Comment:
Somehow admin users can't log in on the django page. Is it possible the logging is causing a caught error or something? I thought this was a server issue but it works on other branches.

Reply:
Do you mean the Django Admin page?

Reply:
Correct. Does this work for you?

Reply:
This was an issue. It has been fixed in the last commit.